app/app.js

```diff
@@ -1,101 +1,16 @@
-angular.module('myApp', ['headerMenu','filterSearch','bookList'])
-.controller('BookCtrl', function($scope) {
+angular.module('myApp', ['headerMenu','filterSearch','bookList','main', 'ui.router', 'bookDetail','book','bookRecomendations','bookComments']);
+angular.module('myApp')
```

Comment:
The next time you touch this file please extract a `main`, `base` or `root` component and just define the angular module here (line 1)